### PR TITLE
Error logs

### PR DESCRIPTION
**Pull request by firewall2142**
Changes:
- error messages are shown in logs
- Emacs `C-c C-c` on comment proves up to the comment
- Diagnostic message won't hide hover result in emacs
- Green zone in VScode won't stick to the cursor

### CODE REVIEW

**Comment by fblanqui** [comment]
Hi @firewall2142 . Thank you very much for your PR! Would you mind to update editors/TODO as well?

**Comment by fblanqui** [comment]
In Emacs, error messages get printed in the logs panel with a useless newline at the beginning, and without position information.

**Comment by fblanqui** [comment]
Other remark: in VSCode, goals are printed even after an error.

**Comment by fblanqui** [comment]
Other remark: in Emacs, no message "No goal" or "Proof finished" is provided at the end of a proof. Instead, there is only a long line.

**Comment by firewall2142** [comment]
I have exams at the moment; I will come back to this PR after some time

**Comment by firewall2142** [comment]
I have exams at the moment; I will come back to this PR after some time

**Comment by fblanqui** [comment]
No problem. Thank you very much Ashish and good luck for your exams!

**Comment by firewall2142** [comment]
@fblanqui I have made the changes.

**Comment by fblanqui** [comment]
Thank you very much @firewall2142 ! Would you mind to add a short-cut for eglot-shutdown as well (C-c C-k ?).

**Comment by fblanqui** [comment]
Thanks Ashish.
You fixed: "When doing C-c C-c on a comment, expand the green until this comment (and not until the following command)." Is it possible to do this when doing C-C C-C on a blank line too?

**Comment by fblanqui** [comment]
Concerning C-c C-r, I get the following error:
```
jsonrpc-error: jsonrpc-error: "No current JSON-RPC connection", (jsonrpc-error-code . 32603), (jsonrpc-error-message . "No current JSON-RPC connection")
```
C-c C-k works well.

File: editors/TODO [hunk 1]
```diff
@@ -1,22 +1,13 @@
 Emacs:
 
-- add shortcut C-c C-r for eglot-reconnect
-
-- When doing C-c C-c on a comment, expand the green until this comment (and not until the following command).
-
 - Do not change the green zone and the goals when editing or adding a comment in the green zone.
 
-- Add buttons for forward/backward navigation.
```
Comment (fblanqui) [inline editors/TODO:9]: I was thinking here about new buttons for C-c C-f/b.

Comment (firewall2142) [reply editors/TODO:9]: Do you mean new icons?

Comment (fblanqui) [reply editors/TODO:9]: Right, new icons. Aren't the current arrow icons for C-c C-n/p?

Comment (firewall2142) [reply editors/TODO:9]: There are also buttons for C-c C-n/p. Try expanding the window.

Comment (firewall2142) [reply editors/TODO:9]: Yes. I took the current icons from `/usr/share/emacs/27.1/etc/images`. It is also possible to add custom icons.

Comment (fblanqui) [reply editors/TODO:9]: In this case, you should check the license/rights and add a editors/emacs/THANKS file.

**Comment by fblanqui** [comment]
@firewall2142 Thanks to update CHANGES.md as well.

File: editors/TODO (hunk 1 above)
Comment (firewall2142) [reply editors/TODO:9]: done

**Comment by fblanqui** [comment]
Thanks @firewall2142 for your PR!

**Pull request merged**